appserver/jms/admin/src/main/java/org/glassfish/jms/admin/cli/JMSDestination.java

```diff
@@ -531,9 +531,10 @@ protected void logAndHandleException(Exception e, String errorMsg)
                                             throws JMSAdminException {
             //log JMX Exception trace as WARNING
             StringWriter s = new StringWriter();
-            e.getCause().printStackTrace(new PrintWriter(s));
+            e.printStackTrace(new PrintWriter(s));
             logger.log(Level.WARNING, s.toString());
```

Comment:
The updated change has avoided changing in the super class - JMSDestination - to reduce the scope of testing necessary.